### PR TITLE
Fix angular2

### PR DESCRIPTION
**Pull request by DeusProx**

### CODE REVIEW

**Comment by DeusProx** [comment]
Funktioniert doch nicht so einfach wegen dem sinnlosen aufbau der systemjs config!

File: systemjs.config.js [hunk 1]
```diff
@@ -7,8 +7,10 @@
   // map tells the System loader where  to look for things
   var map = { 
     'src':                        'app' , // 'dist',
-    '@angular':                   'https://npmcdn.com/@angular',
-    '@angular2-material':         'https://npmcdn.com/@angular2-material',
+    '@angular/router':            'https://npmcdn.com/@angular/router',
+    '@angular/router-deprecated': 'https://npmcdn.com/@angular/router-deprecated',
+    '@angular':                   'https://discussify.github.io/node_modules/@angular',
```
Comment (DadaMonad) [inline systemjs.config.js]: Better use a relative path, righty right?

File: systemjs.config.js [hunk 2]
```diff
@@ -7,8 +7,8 @@
   // map tells the System loader where  to look for things
   var map = { 
     'src':                        'app' , // 'dist',
-    '@angular':                   'https://npmcdn.com/@angular',
-    '@angular2-material':         'https://npmcdn.com/@angular2-material',
+    '@angular':                   'https://discussify.github.io/node_modules/@angular',
```
Comment (DadaMonad) [inline systemjs.config.js]: Better use a relative path, righty right?

**Comment by DeusProx** [comment]
zone.js:101 GET https://discussify.github.io/node_modules/@angular/router//bundles/router.umd.js 404 (Not Found)

still he tries to get the wrong file.... i hate angular2 for trying to use 3 different routers

**Comment by DeusProx** [comment]
works... damn router package had no bundle!

File: systemjs.config.js (hunk 2 above)
Comment (DeusProx) [inline systemjs.config.js]: yeah... you are right^^ but it also works this way xD

**Comment by DadaMonad** [comment]
aaaah the new router package of course xD

**Pull request merged**